.github/workflows/ci.yml

```diff
@@ -102,15 +102,12 @@ jobs:
         run: check/nbformat
   cirq-only:
     name: Pytest (cirq-only) Ubuntu
-    strategy:
-      matrix:
-        python-version: [ '3.6', '3.8' ]
     runs-on: ubuntu-16.04
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v1
         with:
-          python-version: ${{ matrix.python-version }}
+          python-version: '3.8'
```

Comment:
Is this sufficient? I guess the assumption here is that a cirq-only python3.8 run + an all-cirq python3.(6|7) run is enough to verify that cirq-only python3.(6|7) should work?

Reply:
Yes, exactly! One more implicit assumption beyond that is that `pytest.importorskip` works similarly across all 3 versions.